src/main/java/org/prebid/server/auction/AuctionRequestFactory.java

```diff
@@ -533,11 +536,14 @@ private Timeout timeout(BidRequest bidRequest, long startTime, TimeoutResolver t
      */
     private Future<Account> accountFrom(BidRequest bidRequest, Timeout timeout) {
         final String accountId = accountIdFrom(bidRequest);
+        final Future<Account> failedAccountResult = isAccountEnforced
```

Comment:
It make sense to extract this to the separate method and call where we have to, so we'll not execute it if not needed.